### PR TITLE
Adding fastnanquantile

### PR DESCRIPTION
**Pull request by lbferreira**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by lbferreira** [comment]
I am willing to be a maintainer of this new package.

**Comment by lbferreira** [comment]
@conda-forge/help-python The recipe has passed all checks and is ready for your review. Thanks.

File: recipes/fastnanquantile/meta.yaml
Comment (marcelotrevisani) [inline recipes/fastnanquantile/meta.yaml:9]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

File: recipes/fastnanquantile/meta.yaml
Comment (marcelotrevisani) [inline recipes/fastnanquantile/meta.yaml:15]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

**Comment by lbferreira** [comment]
Hi @marcelotrevisani. Thanks for reviewing my PR.
Your suggestions were addressed in the last commit and all tests are passing.

**Comment by lbferreira** [comment]
@conda-forge/help-python The recipe has passed all checks and is ready for your second review. Thanks.

File: recipes/fastnanquantile/meta.yaml
Comment (xhochy) [inline recipes/fastnanquantile/meta.yaml:20]: See the linter message, you probably need setuptools here.

File: recipes/fastnanquantile/meta.yaml
Comment (xhochy) [inline recipes/fastnanquantile/meta.yaml:26]:
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check
```

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/fastnanquantile/meta.yaml
Comment (lbferreira) [reply recipes/fastnanquantile/meta.yaml:19]: Added.

File: recipes/fastnanquantile/meta.yaml
Comment (lbferreira) [reply recipes/fastnanquantile/meta.yaml:27]: This suggestion was addressed.

**Comment by lbferreira** [comment]
@xhochy Thanks for your review!
All your suggestions were addressed and all tests are passing. This PR is ready for your review.

**Review by xhochy — APPROVED** [review]

**Pull request merged**
